modules/tinymce/src/core/main/ts/api/html/SaxParser.ts

```diff
@@ -451,7 +459,10 @@ function SaxParser(settings?: SaxParserSettings, schema = Schema()): SaxParser {
             attrList = [];
             attrList.map = {};
 
-            attribsValue.replace(attrRegExp, parseAttribute);
+            attribsValue.replace(attrRegExp, (match, name, val, val2, val3) => {
+              parseAttribute(value, match, name, val, val2, val3);
+              return '';
```

Comment:
This was needed as `replace` needs a string returned, but `parseAttribute` returns `void` so this wasn't technically ever valid previously. It's only flagged now, as I added what the types were.

Reply:
I should also add we could pull this out, but that's more code and going to be a bit slower (SaxParser we want as fast as possible).